docs/reference/search/search-your-data/search-application-api.asciidoc

```diff
@@ -215,31 +215,35 @@ PUT _application/search_application/my-search-app
       "lang": "mustache",
       "source": """
       {
-        "sub_searches": [
-          {{#text_fields}}
-          {
-            "query": {
-              "match": {
-                "{{.}}": "{{query_string}}"
-              }
-            }
-          },
-          {{/text_fields}}
-          {{#elser_fields}}
-          {
-            "query": {
-              "text_expansion": {
-                "ml.inference.{{.}}_expanded.predicted_value": {
-                  "model_text": "{{query_string}}",
-                    "model_id": "<elser_model_id>"
+        "retriever": {
```

Comment:
One note for search applications - it's possible that people may have defined search applications with templates that use sub_searches (especially since we provided this as an example). Do we want to provide any type of warning to users via search responses that this is deprecated?

Reply:
Good question! Let me follow up on this. Since rank and sub_searches aren't supported on serverless we can do this as a follow up.